crmd/lrm.c

```diff
@@ -84,7 +84,14 @@ make_stop_id(const char *rsc, int call_id)
 
     op_id = calloc(1, strlen(rsc) + 34);
     if (op_id != NULL) {
+#ifdef GCC_FORMAT_TRUNCATION_CHECKING_ENABLED
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-truncation=2"
+#endif
         snprintf(op_id, strlen(rsc) + 34, "%s:%d", rsc, call_id);
+#ifdef GCC_FORMAT_TRUNCATION_CHECKING_ENABLED
+#pragma GCC diagnostic pop
+#endif
```

Comment:
This is not worth the damage to readability. Most of these should use crm_strdup_printf() anyway.